src/main/java/io/jenkins/plugins/jfrog/ReleasesInstaller.java

```diff
@@ -57,5 +49,11 @@ public String getDisplayName() {
         public boolean isApplicable(Class<? extends ToolInstallation> toolType) {
             return toolType == JfrogInstallation.class;
         }
+
+        @POST
+        @SuppressWarnings("unused")
+        public FormValidation doCheckVersion(@QueryParameter String version) {
```

Comment:
doCheckVersion --> doValidateVersion

Reply:
The "doCheck[varialbe]" is a convention in Jenkins forms and can't be changed